python/cudf/cudf/_lib/pylibcudf/groupby.pyx

```diff
@@ -253,26 +256,31 @@ cdef class GroupBy:
         Parameters
         ----------
         values : Table, optional
-            The columns to get group labels for. If not specified, the group
-            labels for the group keys are returned.
+            The columns to get group labels for. If not specified,
+            `None` is returned for the group values.
 
         Returns
         -------
         Tuple[Table, Table, List[int]]
             A tuple of tables containing three items:
                 - A table of group keys
-                - A table of group values
+                - A table of group values or None
                 - A list of integer offsets into the tables
```

Comment:
One last thought: if we returned a pair instead of a triple when values is None, we would probably reorder these outputs to be keys, offsets, values so that the first two outputs would always be the same. Do we still want to do that even if we do return None, such that it's the last return value that gets discarded? `keys, offset, _ = ...` seems more natural than `keys, _, offsets = ...` if we do have to discard an object (or if you're indexing, using a contiguous index set `res[0]` and `res[1]`).

Reply:
That makes sense. I've reordered to `(offsets, keys, values)` as it feels awkward to separate `keys` and `values`.